### PR TITLE
Stable10 migrations

### PR DESCRIPTION
**Pull request by VicDeo**
:exclamation:  this PR adds dependency on **ownCloud 10.0.3+**

Backport of #169

### CODE REVIEW

**Comment by PVince81** [comment]
@VicDeo please fix failures on travis

**Comment by PVince81** [comment]
restarted build

**Comment by VicDeo** [comment]
@PVince81 
the error is
>App "Antivirus App for files" cannot be installed because the following dep  
  endencies are not fulfilled: ownCloud 10.0.3 or higher is required.

**Comment by PVince81** [comment]
ok, I'll increase the version on master...

File: appinfo/install.php [hunk 1]
```diff
@@ -1,11 +1,4 @@
 <?php
 
-$app = new \OCA\Files_Antivirus\AppInfo\Application();
```
Comment (PVince81) [inline appinfo/install.php]: assuming this got removed on purpose, obsolete ?

Comment (VicDeo) [inline appinfo/install.php]: @PVince81 moved into migration.
install script is triggered before migrations, thus here we were trying to insert default rules into the table that did not exist.

**Pull request merged**